public/app/plugins/datasource/tempo/VariableQueryEditor.test.tsx

```diff
@@ -55,7 +55,14 @@ describe('TempoVariableQueryEditor', () => {
     expect(onChange).not.toHaveBeenCalled();
     render(<TempoVariableQueryEditor {...props} onChange={onChange} />);
 
+    // Select the query type ('Label values')
     await selectOptionInTest(screen.getByLabelText('Query type'), 'Label values');
+    await userEvent.click(document.body);
+
+    // Ensure the Label field is shown after selecting the query type
```

Comment:
What would add more context here is explaining why the waitFor is needed. Like we have to wait for an async operation for example.